tiledb/sm/storage_manager/context.h

```diff
@@ -111,27 +105,63 @@ class Context {
   /** The class logger. */
   shared_ptr<Logger> logger_;
 
+  /** The storage manager. */
+  StorageManager* storage_manager_;
+
   inline static std::atomic<uint64_t> logger_id_ = 0;
 
   /* ********************************* */
   /*         PRIVATE METHODS           */
   /* ********************************* */
 
   /**
-   * Initializes the thread pools.
+   * Initializes the context with the input config. Called only from
+   * constructor.
+   */
+  Status init(const Config& config);
+
+  /**
+   * Shutdown and delete the storage manager;
+   */
+  void cleanup_storage_manager();
```

Comment:
Likewise for this function. If I'm reading it correctly, it's only called in the destructor, so it's code should instead just be the destructor.

Reply:
It's called in two places -- the constructor and the destructor.